unitTesting/unit_test_inverse_dynamics_balance_controller.py

```diff
@@ -0,0 +1,54 @@
+from dynamic_graph.sot.torque_control.tests.test_control_manager import cm
+from dynamic_graph.sot.torque_control.tests.robot_data_test import initRobotData
+from dynamic_graph.sot.core.matrix_util import matrixToTuple, vectorToTuple,rotate, matrixToRPY
+from dynamic_graph.sot.torque_control.inverse_dynamics_balance_controller import *
+from numpy import matrix, identity, zeros, eye, array, pi, ndarray, ones
+
+# Instanciate the free flyer
+idbc = InverseDynamicsBalanceController("idbc_test")
+
+q=zeros(initRobotData.nbJoints+6)
+dq=zeros(initRobotData.nbJoints)
+ddq=zeros(initRobotData.nbJoints)
+qRef=zeros(initRobotData.nbJoints+6)
+dqRef=zeros(initRobotData.nbJoints)
+ddqRef=zeros(initRobotData.nbJoints)
+
+# Setting the robot configuration
+idbc.posture_ref_pos.value=qRef
+idbc.posture_ref_vel.value=dqRef
+idbc.posture_ref_acc.value=ddqRef
+
+# Com ref
+comRefPos=zeros(3)
+comRefVel=zeros(3)
+comRefAcc=zeros(3)
+idbc.com_ref_pos.value=comRefPos
+idbc.com_ref_vel.value=comRefVel
+idbc.com_ref_acc.value=comRefAcc
+
+# Right foot
+rfRefPos=zeros(12)
```

Comment:
The first 9 values of this vector represent a rotation matrix, so you should not set all elements to zero.